### DISCUSSION TITLE
Autotilt current mesh?

### DISCUSSION THREAD

**Discussion by BrainNSFW** [Jyers/Marlin#1203] [Q&A]
A quick question regarding the firmware: in the leveling menu, I see an option "autotilt current mesh". What does this do?

EDIT: giving some more info as to why I'm asking.

As in, when do I need to use this feature? Should I use it always after creating a new mesh?

Because right now, my first layers aren't sticking because the nozzle is either too high or too low. Specifically, if I set my nozzle z offset correctly for the center, it will be too high at the outer edges of my bed. If I adjust my z offset for the outer edges, the nozzle scrapes the bed once it moves to the center. I can see that the z motor is working during the print, so I can only assume the (non tilted) mesh is doing something, but apparently it's not enough. I must be missing something, but since no guides on bed lvling mention the auto tilt mesh function, I'm unsure if it will help me here or not.

**Comment by Cal-LF** [comment]
If you Google Autotilt current mesh, several explanations will be given. Try Google first, friendly suggestion.

**Reply by BrainNSFW** [reply]
Obviously I did that first, but I'm not seeing any results actually explaining what it does. As in, when do I need to use this feature? Should I use it always after creating a new mesh? 

Because right now, my first layers aren't sticking because the nozzle is either too high or too low. Specifically, if I set my nozzle z offset correctly for the center, it will be too high at the outer edges of my bed. If I adjust my z offset for the outer edges, the nozzle scrapes the bed once it moves to the center. I can see that the z motor is working during the print, so I can only assume the (non tilted) mesh is doing something, but apparently it's not enough. I must be missing something, but since no guides on bed lvling mention the auto tilt mesh function, I'm unsure if it will help me here or not.

**Reply by Cal-LF** [reply]
Better understanding your question. There are other users that know better how to use the mesh level/tilt feature.
I use the manual mesh with good results.
